### PR TITLE
Adding pfabry

### PR DESCRIPTION
**Pull request by pfabry**

### CODE REVIEW

**Comment by pfabry** [comment]
Adding Paul Fabry infos.

File: _data/operations.yml [hunk 1]
```diff
@@ -278,3 +278,12 @@ members:
...
   wikidata: Q57023310
+- affiliation: Université de Sherbrooke, Québec
+  country: Canada
+  github: pfabry
+  groups:
+  - technical
+  name: Paul Fabry
+  orcid: 0000-0002-3336-2476
+  ror: 00kybxq39
+  wikidata: Q2579532
```
Comment (rays22) [inline _data/operations.yml:289]:
```suggestion
  wikidata: Q108555457
```
The other wikidata entries in this file appear to reference persons, not institutions. For consistency, could you kindly use Q108555457 here?

**Review by rays22 — CHANGES_REQUESTED** [review]

File: _data/operations.yml (hunk 1 above)
Comment (cthoyt) [reply _data/operations.yml:289]: I thought this was clear but maybe we should group the name, location, and ROR for the affiliation in a sub-dictionary in a future PR.

Comment (matentzn) [reply _data/operations.yml:289]: @pfabry can you confirm/apply this suggestion?

**Review by rays22 — APPROVED** [review]

**Pull request merged**